MouseSettings: add ability to configure double-click speed.

thatdutchguy opened the pull request:
Also adds labels to display the values for to each slider.

Before:
![MouseSettings-before](https://user-images.githubusercontent.com/209150/111848684-e878ce00-88c8-11eb-8efe-eb4a1fdcebf2.png)

After:
![MouseSettings-after](https://user-images.githubusercontent.com/209150/111848689-ed3d8200-88c8-11eb-9fe5-25c4c5d6a6c1.png)

ADKaster commented on Userland/Applications/MouseSettings/main.cpp:
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Idan Horowitz <idan.horowitz@gmail.com>
+ * Copyright (c) 2020-2021, the SerenityOS developers.
```
Kinda gauche to remove someone else's copyright, I think you meant to add The SerenityOs developers 2021 below this like?

thatdutchguy replied:
Totally - I did not mean to stomp over that!

thatdutchguy replied:
Updated the license files so credit is given where due.

linusg replied:
Please drop the trailing commas though, this is how it's suppose to look:

https://github.com/SerenityOS/serenity/blob/0a911178cea4fcfb462eb7e256c51ed75756c7c8/Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp#L2-L4

thatdutchguy replied:
What about the single trailing dot at the end of "the SerenityOS deverlopers."? I've noticed that it's present most of the time.

thatdutchguy replied:
Dropped the dot as well, it's now following this style:
https://github.com/SerenityOS/serenity/blob/0a911178cea4fcfb462eb7e256c51ed75756c7c8/Userland/Libraries/LibCompress/Deflate.h#L2-L3

thatdutchguy commented:
Noticed some small UI update issues that I will address when I get a chance later today.

thatdutchguy commented:
All good & rebased on top of master

awesomekling commented on Userland/Services/WindowServer/WindowManager.cpp:
```diff
@@ -172,6 +172,20 @@ void WindowManager::set_scroll_step_size(unsigned step_size)
 
+void WindowManager::set_double_click_speed(int speed)
+{
+    VERIFY(speed >= double_click_speed_min && speed <= double_click_speed_max);
```
This allows anyone to crash the window server by connecting to it and setting an out-of-range doubleclick speed.
Instead of `VERIFY`, we should validate the value and reject invalid ones.

awesomekling replied:
Oh wait nevermind, you already do this in the ClientConnection handler :)

awesomekling commented:
This seems fine, but the commit message is deceptively small: you are adding new IPC API to WindowServer  yet the message only mentions the MouseSettings app. Please make a little more effort to explain what the commit is doing :)

thatdutchguy commented:
Updated commit message. Funny, before I squashed it it had some pretty lengthy comments which seemed a little verbose. At the time it seemed so obvious ;-)
